docs/olx.rst

```diff
@@ -0,0 +1,8 @@
+OLX Conventions
+===================
+
+* Nested XBlocks in separate OLX files can be referenced by adding a **path** attribute to the element which points
+  to the relevant OLX file. The path can be to an OLX file in the same Bundle or in another Bundle.
+* The path to an OLX file may be used as a component of the XBlock ID. Therefore moving an OLX file or changing its
+  name will be equivalent to deleting the XBlock and creating a new one. A block element in an OLX file may
+  also have an **id** attribute. The value of this attribute will be used when constructing the **block_key**.
```

Comment:
@bradenmacdonald If you have a better explanation for this, let me know and I'll edit it in.

Reply:
@symbolist This is fine but it's not very clear what will happen since it says "the path..._may_ be used". It would be better if we can decide for sure how things will be used. Although I'm personally still not sure of the best approach, so this may be a WIP that we update as we try things out.